app/src/main/java/com/swent/assos/ui/login/SignUpScreen.kt

```diff
@@ -41,7 +41,7 @@ fun SignUpScreen(navigationActions: NavigationActions) {
   var confirmPassword by remember { mutableStateOf("") }
   val loginViewModel: LoginViewModel = hiltViewModel()
   var error by remember { mutableStateOf("") }
-  var passwordTooShort by remember { mutableStateOf(false) }
+  var badCredentials by remember { mutableStateOf(false) }
```

Comment:
Same as before, create it in the `viewModel`and set its value in the `signUp`method.